src/core/factory/NotifierFactory.ts

```diff
@@ -0,0 +1,4 @@
+import { NotifierPort } from "core/application/port";
+import { useNotifier } from "core/infrastructure";
+
+export const makeNotifier = (): NotifierPort => useNotifier();
```

Comment:
You do not need to export this function, in your code, you use directly `useNotifier`, it's good.